styles/styles.css

```
@@ -0,0 +1,345 @@
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/ptsans-400.woff2") format(woff2);
```

Название формата нужно заключить в кавычки.
```suggestion
    src: url("../fonts/ptsans-400.woff2") format("woff2");
```